pkg/local_object_storage/blobstor/common/storage.go

```diff
@@ -69,15 +78,32 @@ func Copy(dst, src Storage) error {
 			return nil
 		}
 
-		err = dst.Put(addr, data)
-		if err != nil {
-			return fmt.Errorf("put object %s into destination sub-storage: %w", addr, err)
+		if batchSize <= 1 {
+			err = dst.Put(addr, data)
+			if err != nil {
+				return fmt.Errorf("put object %s into destination sub-storage: %w", addr, err)
+			}
+			return nil
+		}
+		objBatch[addr] = data
```

Comment:
iinm `objBatch` can be non-empty after `src.Iterate` and we have to finish recording